app/services/workflow/updated_event.rb

```diff
@@ -1,6 +1,7 @@
 module Workflow
   class UpdatedEvent
     def self.call(target:, **)
+      return unless target.is_a?(ScannedResource) || target.is_a?(MultiVolumeWork)
```

Comment:
Do we want to maintain the list of models here, or add something to the models that should generate events?